test/test_loaders.py

```diff
@@ -38,6 +40,13 @@ def test_base_url_override(openapi3_schema_url, url):
     assert endpoint.base_url == "http://example.com"
 
 
+def test_port_override(openapi3_schema_url):
+    schema = schemathesis.from_uri(openapi3_schema_url, port=8080)
+    endpoint = next(schema.get_all_endpoints())
+    url = re.sub(r':(\d+)$', '', urlparse(openapi3_schema_url).netloc)
+    assert f"{url}:8080" in endpoint.base_url
```

Comment:
For simplicity, I suggest `assert endpoint.base_url == "http://127.0.0.1:8081/schema.yaml"` (later I'll add a separate function to avoid hardcoding the address)